plugins-compat-tester-cli/src/main/java/org/jenkins/tools/test/CliOptions.java

```diff
@@ -83,6 +83,13 @@
                     "If not set, see includePlugins behaviour.")
     private String excludePlugins = null;
 
+    @Parameter(names = "-alternativePluginOrganization",
+            description = "Include an alternative organization to use as a fallback to download the plugin.\n" +
```

Comment:
Would it make sense to Rename it to 'fallbackGitHubOrganization' to be more explicit?

Reply:
Changed!!